.rubocop.yml

```diff
@@ -2,35 +2,37 @@
 # To see all cops used see here: https://github.com/bbatsov/rubocop/blob/master/config/enabled.yml
 
 AllCops:
-  TargetRubyVersion: 2.4
   DisplayCopNames: true
-
-
-LineLength:
-  Max: 120
+  TargetRubyVersion: 2.4
```

Comment:
Please merge master to your branch to keep it clean and make sure we will not replace original content of this file in master